### PR TITLE
chore: add an issue template.

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

**Comment by cironunes** [comment]
CI failing, probably a flakiness in Apveyor. Can you try to restart it?

**Comment by Brocco** [comment]
I love this idea, my only concern is that it may be too verbose and therefore could overwhelm someone who wants to help

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+> > build your awesome web applications!
+>
+> ----
+>
+> I want to help!
+>
+> > Please read our contributing guidelines;
+> > https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md
+>
+> > Do not file an issue. These will be closed without an explanation. If
```
Comment (filipesilva) [inline .github/ISSUE_TEMPLATE.md]: I get what you're saying here, but the `closed without an explanation` sounds just a bit hostile.

If people want to help, I think the last bit of this paragraph is enough (from "If (instead) you really want...").

**Comment by filipesilva** [comment]
Yeah I'm kinda with Mike on this. It is quite verbose... Maybe we should leave in the first section, and add the others to the readme? 

I like what they say but don't think we get enough issues like those to warrant so much text on the issue template.

**Comment by hansl** [comment]
Okay I removed some lines and simplified others. I think most people will want to file issues so really they shouldn't read past the first section.

**Comment by Brocco** [comment]
Thanks for updating @hansl lgtm

**Pull request merged**